### PR TITLE
Help Required On Switches

### PR DESCRIPTION
**Pull request by jjanicefong**

### CODE REVIEW

**Comment by josephbondoc** [comment]
You can set your buttons to change the stylesheet which would be a duplicate of the original styles 

Except each style has the different width value

:)

**Comment by sshenis** [comment]
Janice! I figured out a way to do the first switch and change its value.

- You need to make a CONST variable and set the state to FALSE so those const [x, y] = useState(z);

- Then MAKE A VARIABLE and set it to the width of the NORMAL STATE so var normalState = x;
- Make an IF ELSE statement:
If (x === false) then the variable is set to the % of the wide switch
else it will be set to the variable width (which is the default state)

and then in your return
make sure the value of your switch is x (for your const)
and the onvaluechange is the y

**Comment by jjanicefong** [comment]
hey @sshenis I implemented the changes you provided but somehow the Wide switch still doesn't expand.

File: comps/Right.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  // REMINDER YOU ARE CODING IN YOUR OWN BRANCH NOT MASTER
+   if (setWidth === false) {
+      normalState = "50%"
+   } else {
+      normalState 
+   }
+  
+  return (
+
+    <View style={styles.right}>
```
Comment (sshenis) [inline comps/Right.js]: Janice, add another style there and set the width as your normalState variable

**Comment by josephbondoc** [comment]
Like the previous nightmode assignment, your switches can be changing the stylesheet each time

Give a variable with the value of "null" (This will be your style change variable!)

ex. var widthChange = null

Since your const is set to (false), you want to give your switch the **DEFAULT** styles when it is FALSE (default width, flex, height etc)

We'll call our default styles "styles.app"

ex. 
const [modeSwitch, setModeSwitch] = useState(false);
   var widthChange = null;
   if(modeSwitch === false){
      widthChange = styles.app (DEFAULT STYLE)

Once it's positive, give it a new style (Which will override the original)
This is when you implement the style with the new width value
DO THIS AS AN ELSE STATEMENT

We'll call the new stylesheet with the new values "styles.width1"

else {
      widthChange = styles.width1
  }

**Your stylesheets should be duplicates of the original EXCEPT a different width value**

**Comment by sshenis** [comment]
Great effort. Thank you for implementing my code and playing around with it until it worked. It's unfortunate we didn't have time to go through the rest but I believe the process for the rest should be the same.

**Pull request merged**